docs/using.rst

```diff
@@ -801,6 +801,27 @@ Example usage for DNS-01 (Cloudflare API v4) (for example purposes only, do not
 
 .. _lock-files:
 
+Changing the ACME Server
+========================
+
+By default, Certbot uses Let's Encrypt's initial production server at
+https://acme-v01.api.letsencrypt.org/. You can tell Certbot to use a
+different CA by providing ``--server`` on the command line or in a
+:ref:`configuration file <config-file>` with the URL of the server's
+ACME directory. For example, to test against Let's Encrypt's ACMEv2
+staging server, you would add ``--server
+https://acme-staging-v02.api.letsencrypt.org/directory`` to the command
+line. Certbot will automatically change the version of the ACME protocol
+used based on the contents served at the provided URL.
+
+If you use ``--server`` to specify an ACME CA that implements a newer
+version of the spec, you may want to obtain a certificate for a wildcard
```

Comment:
you may want to --> you will be able to

Reply:
I used "you may be able to" because it depends on the CA's policy.

Reply:
cool

Reply:
cool